docs/Administrator-Guide/Split-brain-and-ways-to-deal-with-it.md

```diff
@@ -17,12 +17,12 @@ Split brain can happen mainly because of 2 reasons:
     - Server1 comes up, server2 goes down (Heal not happened / data on server 2 is not replicated on server1): Writes happen on server1.
     - Server2 comes up: Both server1 and server2 has data independent of each other.
 
-If we use the replica 2 volume, it is not possible to prevent split-brain without losing availability.
+If we use the "replica 2" volume, it is not possible to prevent split-brain without losing availability.

```

Comment:
using `replica 2` would be a better approach than quoting it with double qoutes

Reply:
Should be fixed.